cmds/basename/basename.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2012-2018 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Basename return name with leading path information removed.
+//
+// Synopsis:
+//     basename NAME [SUFFIX]
+//
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+	"strings"
+)
+
+func init() {
+	// Usage Definition
+	defUsage := flag.Usage
+	flag.Usage = func() {
+		os.Args[0] = "basename NAME [SUFFIX]"
+		defUsage()
+	}
+}
+
+func main() {
+	flag.Parse()
+	if len(flag.Args()) < 1 {
```

Comment:
Make this condition be: `if flag.NArgs() != 1 && flag.NArgs() != 2 {`

Reply:
also can you change this to use the pflag package as in rm/rm.go

Reply:
consider just using a case here ... it's a bit nice looking. See #794.
Also, it's recommended, instead of using flag.Args() everywhere (it's not cheap) to just do something like
a := flag.Args() 
and use that.